scripts/schema/ConvertGGDMv30Schema.py

```diff
@@ -518,6 +518,36 @@ def printAttributeCsv(schema):
 # End printAttributeCsv
 
 
+# Dump the enumerated attributes into files.
+# Each file is named: <attribute>_<FCODE>
+# The files are put in a directory called: enumGGDM30
+def dumpEnumerations(schema):
+    if not os.path.exists('enumGGDM30'):
+        os.makedirs('enumGGDM30')
+    else:
+        shutil.rmtree('enumGGDM30')
+        os.makedirs('enumGGDM30')
+
+    currentDir = os.getcwd()
+    os.chdir('enumGGDM30')
+
+    fCode = ''
+    for i in schema:
+        fCode = schema[i]['fcode']
+
+        for j in schema[i]['columns']:
+            if schema[i]['columns'][j]['type'].find('numeration') > -1:
+                    outFile = open(j + '_' + fCode,'w')
+                    
+                    for k in schema[i]['columns'][j]['enum']:
+                        outFile.write('{} = {}\n'.format(k['name'],k['value']))
+                    
+                    outFile.close()
+
+    os.chdir(currentDir)
+# End dumpEnumerations
+
+
```

Comment:
Since this function is essentially duplicated multiple times, it ought to be put into a separate file.  If you put it in `$HOOT_HOME/scripts/schema/dumpenums.py` or something like that you can add the following at the beginning of each file and use the function the same way as you do in the file.
```
from dumpenums import dumpEnumerations
```
Then you ought to add the directory name as a second parameter to the function:
```
def dumpEnumerations(schema, dirName):
    if not os.path.exists(dirName):
        os.makedirs(dirName)
...
```
...so you call it like this:
```
elif args.dumpenum:
    dumpEnumerations(schema, 'enumGGDM30')
```

Reply:
Done.

I move a lot of functions into a hootLibrary.py file.